config/locales/find.yml

```diff
@@ -141,6 +141,8 @@ en:
           html: <abbr title='Postgraduate certificate in education'>PGCE</abbr> without <abbr title='Qualified teacher status'>QTS</abbr>
         pgde:
           html: <abbr title='Postgraduate diploma in education'>PGDE</abbr> without <abbr title='Qualified teacher status'>QTS</abbr>
+    links:
+      url_national_pay_scales: https://www.gov.uk/government/publications/national-pay-scales-for-eligible-teaching-and-education-jobs/national-pay-scales-for-eligible-teaching-and-education-leadership-occupation-codes
```

Comment:
I think we could drop the `url` here since it's under the links section, but not a blocker. 

So it would just be `national_pay_scales`